src/plugins/azgenerator/CodeModelAzImpl.ts

```diff
@@ -263,7 +267,7 @@ export class CodeModelCliImpl implements CodeModelAz
 
     public get Command_Name(): string
     {
-        this.session.message({Channel:Channel.Warning, Text: "operationGroupIndex: " + this.currentOperationGroupIndex + " operationIndex: " + this.currentOperationIndex});
+        //this.session.message({Channel:Channel.Warning, Text: "operationGroupIndex: " + this.currentOperationGroupIndex + " operationIndex: " + this.currentOperationIndex});
         return this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].language['az'].command;
```

Comment:
Please remove if it's not needed, same for other commented code. thanks.

Reply:
removed.